<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartCabinetClass</class>
 <widget class="QMainWindow" name="SmartCabinetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1597</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCabinet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{
background-color:rgb(61,149,217);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QComboBox {
border: 1px solid rgb(200, 200, 200);
padding-left:30px;
background-color: rgb(18, 45, 177);
border-radius:5px;
font: 24pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
min-height:90px;
min-width:200px;
}

QComboBox:hover {
background-color: rgb(18, 45, 177);
border-radius:5px;
font: 24pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
}
QComboBox:pressed {
/*background-color: rgba(43, 102, 150,0.9);*/
}

QComboBox QAbstractItemView{
font: 24pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
background-color: rgb(181, 208, 214);
selection-color:rgb(186, 186, 186);
selection-background-color:rgba(56, 155, 173,0.9);
outline:0px;
min-height:100px;
border:1px solid  #339933;
margin: 0 0 0 0;
}

QComboBox QAbstractItemView::item {
font: 24pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
min-height:100px;
border:none;
}

QComboBox QAbstractItemView::item:hover {
font: 24pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
background-color: rgb(181, 208, 214);
min-height:100px;
border:none;
}

QComboBox::drop-down {
image: url(:/SmartCabinet/img/downarrow.png);
width:30px;
subcontrol-origin: padding;
subcontrol-position: top right;
}

QPushButton
{
border: 1px solid rgb(200, 200, 200);
background-color: rgb(18, 45, 177);
border-radius:5px;
font: 24pt &quot;Microsoft YaHei&quot;;
color: rgb(255, 255, 255);
min-height:90px;
min-width:200px;
}

QPushButton:hover
{
border-radius:5px;
background-color: rgb(18, 45, 177);
font: 24pt &quot;Microsoft YaHei&quot;;
color: rgb(255, 255, 255);
}
QPushButton:pressed{
background-color: rgb(43, 102, 150);
}
QPushButton:disabled{
background-color: rgb(173, 173, 173)
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,7,1,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_top" native="true">
      <property name="styleSheet">
       <string notr="true">#label_logo{
border-image: url(:/SmartCabinet/img/logo_32_32.png);
}
#pushButton_exit{
border-image: url(:/SmartCabinet/img/logo2_lu3D.png);
background-color: rgba(255, 255, 255,0);
}
#label_yunan{
font:16pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_logo">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_title">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;智云微库&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>262</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_yunan">
         <property name="text">
          <string>运安数据显示</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_info" native="true">
         <property name="styleSheet">
          <string notr="true">QLabel{
font:16pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
}
QComboBox{
min-height:70px;
min-width:220px;
}
QComboBox#comboBoxUserRole QListView::item::hover
{
background-color: rgba(44, 110, 201,0.7);
}
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayoutBaseInfo" stretch="1,1,1,1,1,1,1,1,18,2,3,0,1,2,1,0,2">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelSpace">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelSpace_2">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCarModel">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;部门：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCarModelValue">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;--&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCarNum">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;职务：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCarNumValue">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;--&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCarSegment">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;车段：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCarSegmentValue">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;--&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelSpace_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUserID">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;用户编号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUserIDValue">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUserRoleValue">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;--&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUserName">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;姓名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUserNameValue">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;--&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUserRole">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;角色：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUserRole_2">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;--&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxUserRole">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>252</width>
              <height>72</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>70</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>24</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>261</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_exit">
         <property name="minimumSize">
          <size>
           <width>202</width>
           <height>92</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_1">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(16, 126, 211,0.6);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ToolsWidget" name="widget_main" native="true"/>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(16, 126, 211,0.6);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_bottom" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_button" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_op" native="true">
         <property name="styleSheet">
          <string notr="true">#label_time{
font: 30pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
padding-left:20%;
}
QComboBox#comboBox_lockType QListView::item::hover
{
background-color: rgba(44, 110, 201,0.7);
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>30</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_time">
            <property name="text">
             <string>currTime</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>539</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_lockType">
            <property name="minimumSize">
             <size>
              <width>232</width>
              <height>92</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>540</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_logout">
            <property name="minimumSize">
             <size>
              <width>202</width>
              <height>92</height>
             </size>
            </property>
            <property name="text">
             <string>退出登录</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ToolsWidget</class>
   <extends>QWidget</extends>
   <header>ToolsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="smartcabinet.qrc"/>
 </resources>
 <connections/>
</ui>
